This test always results in success, just like Python.
